rfcs/0010-replace-oppressive-terminology.md

```diff
@@ -15,29 +15,60 @@ governing the SuperCollider project:
 > A primary goal of our community, our user groups and mailing lists, symposia and other events is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all.
 
 To that end, we should, when possible and practicable, replace usage of oppressive and exclusive language with inclusive
-and accurate alternatives.
+and accurate alternatives. This ensures that for a diverse set of new users, SuperCollider will make them feel welcome
+when reading the documentation and source code.
 
 A full motiviation and argumentation for this change, along with examples of precedent, can be found in the [Internet
 Engineering Task Force](https://tools.ietf.org/id/draft-knodel-terminology-00.html) RFC on the same topic. This RFC
-itself should be regarded as an effective "resubmission" of the IETF RFC for the SuperCollider community; I believe it
-makes the case very well, I agree with it completely, and I have nothing to add to it.
+itself should be regarded as an effective "resubmission" of the IETF RFC for the SuperCollider community.
 
 # Specification
 
-(Much of this is directly adapted from the IETF RFC above)
-
 We should:
 - **Change the 'master' branches of repositories 'master' to 'main'**, as shown in https://www.hanselman.com/blog/EasilyRenameYourGitDefaultBranchFromMasterToMain.aspx.
 - Prevent new commits from being pushed to 'master' branches of repositories.
-- Replace the oppressive terms “master-slave” with more accurate alternatives.
+- Keep inactive 'master' branches on repos where they currently exist, for historical purposes.
+- For new repos in the SuperCollider organization which are not being forked, name the default branch 'main' and do not
+  create a 'master' branch.
+- Replace terms using 'master' with 'main', for example 'master volume' to 'main volume'.
+- Replace the oppressive terms “master-slave” with more accurate alternatives:
+    - primary-secondary
+    - conductor-follower
+    - active-standby
+    - primary-replica
+    - writer-reader
+    - parent-helper
+    - coordinator-worker
 - Replace the oppressive terms “blacklist-whitelist” with more accurate alternatives.
+    - blocklist-allowlist
```

Comment:
blocklist-passlist ?

Reply:
as you asked for a rationale, here it is: 

"blocking" and "passing" are of the same semantic category (physical action), as "forbidding" and "allowing" would be (moral/legal), but not "blocking" and "permitting". This is why I looked for something that rhymes better.

The same applies to block-permit, of course.

Reply:
thank you for the rationale! I am still keeping it to these terms for the reasons I gave below. at least in this point in time, a quick internet search shows that momentum is behind blocklist/allowlist as replacements.

I don't agree with the semantic argument either. these semantic categories you gave don't match with my understanding of these terms. "block" is used quite often when, for example, a book or law or legal order is prevented from taking effect. it has a clear meaning in legal contexts. to give a recent example from the news -- ["The European Union is expected to block most US residents from visiting the region when travel restarts due to rising case numbers. The EU is finalising the list of visitors allowed into the bloc as early as July. "](https://www.news.com.au/world/americans-unlikely-to-be-let-into-eu/video/b537c9abc557c66fb8b393384c69a2cc). note the pairing "block from" / "allow into".

Reply:
OK, I do see the point, although the alternative seems still clearer. 

In any case, in the linux kernel it is now:
```
+Recommended replacements for 'blacklist/whitelist' are:
+    'denylist / allowlist'
+    'blocklist / passlist'
```

https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=49decddd39e5f6132ccd7d9fdc3d7c470b0061bb